### DISCUSSION TITLE
deepxde for images

### DISCUSSION THREAD

**Discussion by UgoB7** [lululxvi/deepxde#1444] [Q&A]
Hello,

I would like to apply DeepXDE for image processing. If I have a 256x256 image and I set num_domain = 256*256 with a geom = dde.geometry.Rectangle([0,0], [1,1]) geometry. Are the points uniform such as I can have as IC my image to process and let DeepXDE resolve the PDE on the image over time ? Have you heard of use of DeepXDE for image proc ?
Cordially,
Ugo

**Comment by lululxvi** [comment]
I don't really understand your question.
